cli/report/report.py

```diff
@@ -373,6 +375,10 @@ def add_passing_job_comment(self, job: Job, jira: Jira, issue_id: str) -> None:
                                 This comment was created using [firewatch in OpenShift CI|https://github.com/CSPI-QE/firewatch].
                             """
         jira.comment(issue_id=issue_id, comment=comment)
+        jira.add_labels_to_issue(
+            issue_id_or_key=issue_id,
+            labels=[JOB_PASSED_SINCE_TICKET_CREATED_LABEL],
```

Comment:
If the `JOB_PASSED_SINCE_TICKET_CREATED_LABEL` variable is only ever going to be used in this method, do you think it should still be defined as a constant above? I'm fine with doing that, just looking for your thoughts.

Reply:
@calebevans It's also imported and used in tests. Keeping it as a constant makes it easier to rename and find all of the usages than using a string literal would. Normally I would extract this and other constants to a `constants.py` file in the same module when refactoring, but Just let me know if you think it should be moved somewhere else or stay where it is.

@Simranpal This method should only ever be called when the job has passed and a "passing" comment needs to be added. [Here is the conditional logic](https://github.com/RedHatQE/firewatch/blob/79274efd28f54da8c9751c56950c4beaac938f57/cli/report/report.py#L82-L92) used for calling the `add_passing_job_comment` method.

>  Is it going to added always?

That's a good question. As it is, yes, it will always be added to that ticket. 

I worded the label to imply that the job has passed at least once since the ticket was created, and to have a very specific meaning. I would like to hear more thoughts on whether we should change it or add other labels for different conditions.

Thanks for the feedback!

Reply:
@amp-rh I think I like the idea of a `constants.py` file. I think it makes more sense, especially if we are going to be importing the value into tests, etc.

I am also wondering if we should rename this method. Or maybe we move the `jira.add_labels_to_issue` line to [the same place that this method is called from](https://github.com/RedHatQE/firewatch/blob/79274efd28f54da8c9751c56950c4beaac938f57/cli/report/report.py#L88)? What do you think?

Reply:
> I am also wondering if we should rename this method. Or maybe we move the jira.add_labels_to_issue line to [the same place that this method is called from](https://github.com/RedHatQE/firewatch/blob/79274efd28f54da8c9751c56950c4beaac938f57/cli/report/report.py#L88)? What do you think?

I agree, that would make more sense since they are at the same level of abstraction. I'll make those changes.